### PR TITLE
unmanaged state

### PR DESCRIPTION
**Pull request by rjastrzebski**

### CODE REVIEW

File: src/vmshepherd/presets/preset.py [hunk 1]
```diff
@@ -81,6 +83,7 @@ def _reset_counters(self):
         """
         if if_older_than is None or time.time() - self._vms_refresh_time > if_older_than:
             self._vms_refresh_time = time.time()
+            print('*'*100, type(self.iaas), self.name)
```
Comment (kwarunek) [inline src/vmshepherd/presets/preset.py]: ?

File: src/vmshepherd/presets/preset.py [hunk 2]
```diff
@@ -10,6 +10,7 @@ class Preset:
     def __init__(self, name: str):
         self.iaas = None
         self.healthcheck = None
+        self.unmanaged = False
```
Comment (kwarunek) [inline src/vmshepherd/presets/preset.py]: I presumed that "unmanaged" is a runtime state, that could be switch on/off via API, trigger by events (unavailability of some part)...

Comment (rjastrzebski) [reply src/vmshepherd/presets/preset.py]: yep, maybe, but this is only implementation of unmanaged fuctionality in static configuration
- preset configuration or config file of vmshepherd

Comment (rjastrzebski) [reply src/vmshepherd/presets/preset.py]: its required  for AWS where presets are managed by ASG.

**Pull request merged**